README.md

```diff
@@ -40,7 +40,7 @@ In the meantime, you can build and run Periskop from source:
 
 ```
 docker build --tag periskop .
-docker run -v path/to/config.yaml:/etc/periskop/periskop.yaml -p 8080:8080 periskop
+docker run -e SERVER_URL=`docker-machine ip default` -e CONFIG_FILE=/etc/periskop/config/<config-file>.yaml -v /path/to/config/yaml/folder:/etc/periskop/config -p 8080:8080 periskop
```

Comment:
I understand the context of this change related to using docker-compose but I'd prefer to keep the old behavior for the production configuration, where the web component is able to call the api on whatever host the former is served from. With that behavior, in addition to reducing operational overload, we were able to support "for free" a setup where there is a proxy in front of Periskop.

Reply:
This change is not related to `docker-compose` (the `SERVER_URL` bit of course is, the mounting of the config file is not, at least when I try to run the "old" command it fails as it says that the config file is a directory.

The `SERVER_URL` can be inferred to `localhost` if not provided. I can remove that for sure.

Reply:
Yeah my comment was actually referring to the `SERVER_URL` bit. 

I didn't notice the change in how you pass the config to the container, I can have a look on why that broke 😅 Just for reference, that syntax is inspired to [the Prometheus one](https://prometheus.io/docs/prometheus/latest/installation/#volumes-bind-mount).

Reply:
@rciovati I removed the `SERVER_URL` part as it shouldn't be needed anymore as it should default to `localhost`